scripts/darwin/package.sh

```diff
@@ -97,8 +93,10 @@ fi
 APPLICATION_OUTPUT="$ARGV_OUTPUT/$ARGV_APPLICATION_NAME.app"
 mv "$ARGV_OUTPUT/Electron.app" "$APPLICATION_OUTPUT"
 rm "$APPLICATION_OUTPUT/Contents/Resources/default_app.asar"
-cp "$ARGV_LICENSE" "$ARGV_OUTPUT/LICENSE"
-echo "$ARGV_VERSION" > $ARGV_OUTPUT/version
+
+# Don't include these for now
+rm "$ARGV_OUTPUT"/LICENSE*
+rm "$ARGV_OUTPUT/version"
```

Comment:
These should probably be `rm -f ` so that they don't fail when the files they're trying to delete don't exist?